lib/connection/connection_config.js

```diff
@@ -363,6 +364,12 @@ function ConnectionConfig(options, validateCredentials, qaMode, clientInfo)
       JSON.stringify(fetchAsString[invalidValueIndex]));
   }
 
+  let rowMode = options.rowMode;
```

Comment:
we don't override it so it should be `const`? or maybe we should have default value for this parameter somewhere in this file and here we only allow user to override it?
If we don't want to have default value let's write a comment that it is `undefined` and it means that we are returning values as an object.

Reply:
fixed